### ISSUE TITLE
REML lists in main display omit text after embedded UL or OL

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/codesnip#82]
If a list is embedded within a list, and text in the outer list item follows the end of the embedded list then that text is not displayed in the main display, unless it is itself a block, such as a paragraph.

For example:

```html
<ol>
  <li>
    Foo
    <ul>
      <li>
        Bar
      </li>
    </ul>
    Baz
  </li>
</ol>
```

will display `Foo` but not `Baz`, but

```html
<ol>
  <li>
    Foo
    <ul>
      <li>
        Bar
      </li>
    </ul>
    <p>
      Baz
    </p>
  </li>
</ol>
```

will display both `Foo` and `Baz`.

**Comment by delphidabbler** [comment]
Working on fixing at same time as issue #81, in the #81 feature branch.

**Comment by delphidabbler** [comment]
Fixed at merge commit d9da938ef60eb9f80b1b70564ecfd2afee27a6a2
